charts/images.yaml

```
@@ -159,17 +145,7 @@ images:
   sourceRepository: github.com/gardener/autoscaler
   repository: eu.gcr.io/gardener-project/gardener/autoscaler/cluster-autoscaler
   tag: "v1.22.6"
-  targetVersion: "1.22.x"
-- name: cluster-autoscaler
-  sourceRepository: github.com/gardener/autoscaler
-  repository: eu.gcr.io/gardener-project/gardener/autoscaler/cluster-autoscaler
-  tag: "v1.21.6"
-  targetVersion: "1.21.x"
-- name: cluster-autoscaler
-  sourceRepository: github.com/gardener/autoscaler
-  repository: eu.gcr.io/gardener-project/gardener/autoscaler/cluster-autoscaler
-  tag: "v1.20.4"
-  targetVersion: "< 1.21"
+  targetVersion: "<= 1.22"
```

```suggestion
  targetVersion: "1.22.x"
```

To prevent confusion since we no longer support < 1.22